crates/database/src/lib.rs

```diff
@@ -13,6 +13,7 @@ pub mod onchain_invalidations;
 pub mod order_events;
 pub mod order_execution;
 pub mod orders;
+pub mod orders_history;
```

Comment:
```suggestion
pub mod order_history;
```

is more correct english I believe (cf. other compound word order mods)